plugins/reference-assets/core/src/assets/action/types.ts

```diff
@@ -27,6 +27,9 @@ export interface ActionAsset<AnyTextAsset extends Asset = Asset>
 
     /** Force transition to the next view without checking for validation */
     skipValidation?: boolean;
+
+    /** boolean value to decide for the left anchor sign */
+    role?: boolean;
```

Comment:
I think we would want this to be `role?: string;` so we can potentially support other roles in the future, but the transform can populate it with a `back` value when appropriate

Reply:
Hey @hborawski , so if I make the role as string then In the transform, I need to check if it is a backButton and then apply the role as 'back'(maybe). In the action component, then check for the role value. Is this understanding correct?

Reply:
yes this is correct, as `"role": true` does not really have any semantic intent, `"role": "back"` is more clear as to what is happening,